packages/components/atoms/f-image-tile/jest.config.js

```diff
@@ -28,7 +28,8 @@ module.exports = {
 
     globals: {
         'vue-jest': {
-            hideStyleWarn: true // We hide style warnings given the first time we run the tests it complains about some styles. The second time the tests are run, the warning disappears. https://github.com/vuejs/vue-jest/issues/178#issuecomment-529175129
+            hideStyleWarn: true, // We hide style warnings given the first time we run the tests it complains about some styles. The second time the tests are run, the warning disappears. https://github.com/vuejs/vue-jest/issues/178#issuecomment-529175129
+            experimentalCSSCompile: false
```

Comment:
Need to remove this before merging ideally but without this in place `vue-jest` complains. Bit more info [here](https://github.com/vuejs/vue-jest/issues/137):

Reply:
After spending a few days looking at this, I think we'll need to keep these in so that we can update to dart-sass. It doesn't cause an issue for the majority of packages – any that it does cause an issue in can temporarily add node-sass to their devDependencies in order to test style specific features.